aptos-move/framework/aptos-token/sources/token_transfers.move

```diff
@@ -5,23 +5,48 @@ module aptos_token::token_transfers {
     use std::error;
     use aptos_std::table::{Self, Table};
     use aptos_token::token::{Self, Token, TokenId};
+    use aptos_framework::account;
+    use aptos_framework::event::{Self, EventHandle};
+
 
     const ETOKEN_OFFER_NOT_EXIST: u64 = 1;
 
     struct TokenTransfers has key {
         pending_claims: Table<TokenOfferId, Token>,
+        offer_event: EventHandle<TokenOfferEvent>,
+        cancel_offer_event: EventHandle<TokenCancelOfferEvent>,
+        claim_event: EventHandle<TokenClaimEvent>,
     }
```

Comment:
we cannot break changes like this, we're doing a compatible upgrade, but we can ignore the existing data structure.